src/components/Header/Header.module.scss

```diff
@@ -0,0 +1,107 @@
+.header {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 24px 370px;
+
+    position: relative;
+    width: 1920px;
+    height: 88px;
```

Comment:
1. don't need to set `flex-direction: column;` `align-items: center;`
2. don't need to set `height: 88px;`, and change padding to `padding: 24px 0px;`
3. don't need to set `width: 1920px;`

Reply:
https://github.com/taodemy/taotalk/blob/457af72abe43f54f6b3c079e56f580a15ec67f7f/src/components/Header/Header.module.scss#L3-L11

Previous codes are modified as suggestions in guidance. And other similar problems in css file are also modified accordingly.